frontend/src/scenes/Settings/components/InviteModal.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { MouseEvent, KeyboardEvent, ChangeEvent, useState } from 'react';
+import { Modal, Form, Button } from 'react-bootstrap';
+import { Routine } from 'redux-saga-routines';
+import { Roles } from '../../../common/enums/UserRoles';
+import styles from './styles.module.scss';
+
+interface IProps {
+  showModal: boolean,
+  clsName: string,
+  setShowModal: React.Dispatch<React.SetStateAction<boolean>>,
+  handleSend: Routine<any>,
+  userChanges: {
+    id?: string,
+    isLoading?: boolean,
+    isFailed?: boolean
+  }
+}
+
+const InviteModal: React.FC<IProps> = ({ showModal, clsName, setShowModal, handleSend, userChanges }) => {
+  const [email, setEmail] = useState('');
+  const [userRole, setRole] = useState(Roles.Viewer);
+  const [emailValid, setEmailValid] = useState(true);
+
+  const handleClose = () => {
+    setEmail('');
+    setRole(Roles.Viewer);
+    setShowModal(false);
+  };
+
+  const isEmailValid = (currentEmail: string) => /(.+)@(.+){2,}\.(.+){2,}/g.test(currentEmail.toLocaleLowerCase());
```

Comment:
Please fix this after we merge sign-in/sign-up frontend functionality. We should use yup lib for validation